src/main/java/org/myrobotlab/framework/interfaces/JsonSender.java

```diff
@@ -0,0 +1,12 @@
+package org.myrobotlab.framework.interfaces;
+
+public interface JsonSender {
+
+  /**
+   * Send interface which takes a json encoded Message.
+   * For schema look at org.myrobotlab.framework.Message
+   * @param jsonEncodedMessage
+   */
+  public void send(String jsonEncodedMessage);
```

Comment:
similar to the JsonInvoker interface, we should have a default implementation here so that specific services don't need to parse this jsonEncodedMessage.